AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -393,33 +394,34 @@ void Tracker::mavlink_snoop(const mavlink_message_t* msg)
     {
         // decode
         mavlink_global_position_int_t packet;
-        mavlink_msg_global_position_int_decode(msg, &packet);
-        tracking_update_position(packet);
+        mavlink_msg_global_position_int_decode(&msg, &packet);
+        tracker.tracking_update_position(packet);
         break;
     }
     
     case MAVLINK_MSG_ID_SCALED_PRESSURE:
     {
         // decode
         mavlink_scaled_pressure_t packet;
-        mavlink_msg_scaled_pressure_decode(msg, &packet);
-        tracking_update_pressure(packet);
+        mavlink_msg_scaled_pressure_decode(&msg, &packet);
+        tracker.tracking_update_pressure(packet);
         break;
     }
     }
+    GCS_MAVLINK::packetReceived(status, msg);
```

Comment:
shouldn't you have a `bool handled` and only call the parent if the packet was **not** handled?

Reply:
I don't think so.  That would preclude the parent doing other funky things with the received packet - most notable updating the routing tables.

Reply:
couldn't that be done by whoever call this function?  then this follows the normal flow of having a default shared handled and allow specializations to do their own if they need. E.g. https://gist.github.com/fdf971c5e377be183fe2babfe12c2b06